### PR TITLE
PMM-11126 (FB)

### PR DESCRIPTION
**Pull request by cezmunsta**
Custom branches: 
[pmm-update](https://github.com/percona/pmm-update/pull/256)

### CODE REVIEW

File: ci.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+deps:
+- name: pmm-update
+  branch: PMM-11126
+  url: https://github.com/percona/pmm-update
```
Comment (ademidoff) [inline ci.yml:4]: You can as well skip the `url` it'll work without it :)

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2910-466c433
client docker - perconalab/pmm-client-fb:PR-2910-466c433
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2910-466c433.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2910-466c433&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2910-466c433.tar.gz

File: ci.yml (hunk 1 above)
Comment (cezmunsta) [reply ci.yml:4]: @atymchuk this is what `make prepare XXX` does :)

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-2910-2505422
client docker - perconalab/pmm-client-fb:PR-2910-2505422
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2910-2505422.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-2910-2505422&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2910-2505422.tar.gz

**Pull request closed without merging**